bin/run_sql_task.py

```diff
@@ -48,12 +48,12 @@ def main():
         task_dir=TASK_ROOT / args.task,
         template_vars={
             "db_user": parse_dsn(dsn)["user"],
+            "metabase_db_user": os.environ.get("MB_DB_USER", "postgres"),
```

Comment:
```suggestion
            "metabase_db_user": os.environ["MB_DB_USER"],
```

I think we are better off insisting on being provided with this and putting it in the `tox.ini`. Otherwise we end up with config potentially coming from all over. We might also paper over a missing variable when deployed.